app/js/components/orderEntry/AllOrders.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import Orders from './Orders';
+
+const AllOrders = () => (
+  <div className="all-orders">
+    <div className="orders-breadcrumb">
+      <h3>Patient Orders</h3>
+    </div>
+    <table className="t-orders">
+      <tbody>
+        <tr>
+          <th className="th-1">Date</th>
```

Comment:
Please use a more readable className for this.

Reply:
ok, I'll address that..

Reply:
addressed